robot_skills/src/robot_skills/robot.py

```diff
@@ -101,6 +119,18 @@ def __init__(self, robot_name="", wait_services=False):
             not_operational_parts = [name for name, part in self.parts.iteritems() if not part.operational]
             rospy.logwarn("Not all hardware operational: {parts}".format(parts=not_operational_parts))
 
+    # Deprecated warning for 'robot.leftArm' to avoid breaking the code.
+    @proprty
```

Comment:
Typo: ```property``` instead of `proprty` (below as well)

Reply:
fixed, along with usn\ing an existing deprecated wrapper